src/Ticket.php

```diff
@@ -4301,7 +4301,7 @@ public function showForm($ID, array $options = [])
             $options['entities_id'] = $item->fields['entities_id'];
         }
 
-        $initial_creation = static::isNewID($ID) && !$this->hasSavedInput();
+        $initial_creation = static::isNewID($ID) && $this->hasSavedInput();
```

Comment:
As it stands, the variable name `$initial_creation` no longer matches its content. So, if this fixes both the case from this PR and the one from #18611, the name should be changed to a more appropriate one.  

If it does not fix both cases, the `if ($initial_creation) { ... }` block further down may need to be reworked to ensure both cases are correctly handled.